openmp/libomptarget/src/api.cpp

```diff
@@ -241,10 +241,125 @@ static int libomp_target_memcpy_async_helper(kmp_int32 Gtid, kmp_task_t *Task) {
   return Rc;
 }
 
+static int libomp_target_memset_async_helper(kmp_int32 Gtid, kmp_task_t *Task) {
+  if (!Task) {
+    return OFFLOAD_FAIL;
+  }
```

Comment:
LLVM style please (no `{}` for single statement.

Reply:
Ack!

Reply:
Resolved via commit 7933bfa0.